AndrewLikesLolis.txt

```diff
@@ -6,4 +6,5 @@ Williamson and Marshall are fighting for Loli Loving King.
 
 George likes lolis.
 
-My commit will be merged before george's!
+Loli lover
```

Comment:
why did you delete my sentence :((((((((((((((((((((((((((

Reply:
Get rekt m8